aspnetcore/release-notes/aspnetcore-7/samples/ApiController/Program.cs

```diff
@@ -0,0 +1,51 @@
+#define MIN // FIRST DIS MIN
+#if NEVER
+#elif FIRST
+#region snippet1
+var builder = WebApplication.CreateBuilder(args);
+
+builder.Services.AddControllers();
+builder.Services.AddSingleton<IDateTime, SystemDateTime>();
+
+var app = builder.Build();
+
+app.MapControllers();
+
+app.Run();
+#endregion
+#elif DIS
+#region snippet_dis
+using Microsoft.AspNetCore.Mvc;
+
+var builder = WebApplication.CreateBuilder(args);
+
+builder.Services.AddControllers();
+builder.Services.AddSingleton<IDateTime, SystemDateTime>();
+
+builder.Services.Configure<ApiBehaviorOptions>(options =>
+{
+    options.DisableImplicitFromServicesParameters = true;
+});
+
+var app = builder.Build();
+
+app.MapControllers();
+
+app.Run();
+#endregion
+#elif MIN
+// The following code works in .NET 6, not new to .NET 7
+#region snippet_min
+using Microsoft.AspNetCore.Mvc;
+
+var builder = WebApplication.CreateBuilder(args);
+builder.Services.AddSingleton<IDateTime, SystemDateTime>();
+
+var app = builder.Build();
+
+app.MapGet("/",   (               IDateTime dateTime) => dateTime.Now);
+app.MapGet("/fs", ([FromServices] IDateTime dateTime) => dateTime.Now);
+app.Run();
+#endregion
+#endif
```

Comment:
Should I mention in .NET 6, minimal APIs don't require `[FromServices]`?

Reply:
> @Rick-Anderson I don't see where this snippet is used in the docs. Are you planning to add something related to that? If not, I am not sure if it is needed

It's not in the doc. I'll remove it. I was wondering if I should say "Minimal APIs in .NET 6 did not require `[FromServices]`

Reply:
In my opinion, we don't need to mention anything here since it is a doc with the new features.